source/puppet/5.3/hiera_automatic.md

```diff
@@ -29,57 +29,60 @@ For example, you can set servers for the `NTP` class like this:
 ntp::servers:
   - time.example.com
   - 0.pool.ntp.org
-  ```
+```
 
-> Note: The best way to manage this is to use the roles and profiles method, which allows you to store a smaller amount of more meaningful data in Hiera. 
+> Note: The best way to manage this is to use the roles and profiles method, which allows you to store a smaller amount of more meaningful data in Hiera.
 
 Related topics: [roles and profiles][r_n_p].
 
-{:.Concept}
-## Puppet lookup  
+{:.concept}
+## Puppet lookup
 
-The `lookup` function uses Hiera to retrieve a value for a given key. 
+The `lookup` function uses Hiera to retrieve a value for a given key.
 
-By default, the lookup function returns the first value found and fails compilation if no values are available. You can also configure the lookup function to merge multiple values into one. 
+By default, the lookup function returns the first value found and fails compilation if no values are available. You can also configure the lookup function to merge multiple values into one.
 
 When looking up a key, Hiera searches up to four hierarchy layers of data, in the following order:
 
-1. Global hierarchy.
-2. The current environment’s hierarchy.
-3. The indicated module’s hierarchy, if the key is of the form `<MODULE NAME>::<SOMETHING>`.
-4. If not found and the module's hierarchy has a default_hierarchy entry in its hiera.yaml - the lookup is repeated if steps 1-3 did not produce a value.
+1.  Global hierarchy.
+2.  The current environment's hierarchy.
+3.  The indicated module's hierarchy, if the key is of the form `<MODULE NAME>::<SOMETHING>`.
+4.  If not found and the module's hierarchy has a default_hierarchy entry in its hiera.yaml - the lookup is repeated if steps 1-3 did not produce a value.
 
 > Note: Hiera checks the global layer before the environment layer. If no global `hiera.yaml` file has been configured, Hiera defaults are used. If you do not want it to use the defaults, you can create an empty `hiera.yaml` file in `/etc/puppetlabs/puppet/hiera.yaml`.
 
 Default global hiera.yaml is installed at /etc/puppetlabs/puppet/hiera.yaml.
 
-### Arguments  
- 
+### Arguments
+
 You must provide the name of a key to look up, and can optionally provide other arguments. You can combine these arguments in the following ways:
-* `lookup( <NAME>, [<VALUE TYPE>], [<MERGE BEHAVIOR>], [<DEFAULT VALUE>] )`
-* `lookup( [<NAME>], <OPTIONS HASH> )`
-* `lookup( as above ) |$key| { <VALUE> } # lambda returns a default value`
- 
+-   `lookup( <NAME>, [<VALUE TYPE>], [<MERGE BEHAVIOR>], [<DEFAULT VALUE>] )`
```

Comment:
I converted unordered lists to use a dash (`-`) instead of an asterisk (`*`). This avoids some ambiguity with bold/italic formatting, which also uses asterisks. I also added two spaces to make them consistent with other four-character list indents.